Units/parser-eiffel.r/obsolete.e.d/input.e

```diff
@@ -1,4 +1,4 @@
-class OBSOLETE obsolete "Use a different one"
+class OBSOLETE_ obsolete "Use a different one"
```

Comment:
I don't understand the intent of the original author of this test input.
1. `OBSOLETE` is an key, so this should not be tagged.
2. `OBSOLETE` is a odd but a class name, so this should be tagged.

If this should not be tagged, we should not change the input. We must keep as is.
If this should be tagged, we have to fix the parser.

I checked the log of subversion but I don't find the original intent.

How do you think?